doc/source/api/remoteappmanager.db.rst

```diff
@@ -37,6 +34,5 @@ Module contents
 
 .. automodule:: remoteappmanager.db
     :members:
-    :special-members: __init__
```

Comment:
These changes on the doc do not seem relevant to this PR, can you create an issue separately please?